.bazelrc

```diff
@@ -23,24 +23,31 @@ build --flag_alias=gcc_version=//:gcc_version
 # Don't include source level debug info on macOS. See
 # https://github.com/urbit/urbit/issues/5561 and
 # https://github.com/urbit/vere/issues/131.
-build:linux --copt='-g'
+build:linux --per_file_copt='pkg/.*@-g'
 build:linux --host_copt='-g'
 build --strip=never
 
 # Use -O3 as the default optimization level.
-build --copt='-O3'
+build --per_file_copt='pkg/.*@-O3'
 build --host_copt='-O3'
 
 # Turn on CPU and memory debug for exec config, which we only use to run the
 # fake ship tests.
 build --host_copt='-DU3_CPU_DEBUG'
 build --host_copt='-DU3_MEMORY_DEBUG'
 
+# Set as per-file copts as a plain --copt gets passed to third party
+# dependencies which forces recompilation (slow) if you are switching between
+# including/excluding a symbol define
+build:mem_dbg --per_file_copt='pkg/.*@-DU3_MEMORY_DEBUG'
+build:cpu_dbg --per_file_copt='pkg/.*@-DU3_CPU_DEBUG'
+
 # Enable maximum debug info and disable optimizations for debug config. It's
 # important that these lines come after setting the default debug and
 # optimization level flags above.
-build:dbg --copt='-O0'
-build:dbg --copt='-g3'
+# build:dbg --copt="-O0"
```

Comment:
Can you remove this line?

Reply:
sorry about that had these changes locally, but didn't upstream them